test/logflare/backends/adaptor/postgres/pipeline_test.exs

```diff
@@ -0,0 +1,90 @@
+defmodule Logflare.Backends.Adaptor.Postgres.PipelineTest do
+  use Logflare.DataCase, async: false
+
+  alias Logflare.Backends
+  alias Logflare.Backends.Adaptor.Postgres
+  alias Logflare.Backends.Adaptor.Postgres.LogEvent
+  alias Logflare.Backends.Adaptor.Postgres.Pipeline
+  alias Logflare.Backends.Adaptor.Postgres.Repo
+  alias Logflare.Buffers.MemoryBuffer
+
+  setup do
+    %{username: username, password: password, database: database, hostname: hostname} =
+      Application.get_env(:logflare, Logflare.Repo) |> Map.new()
+
+    url = "postgresql://#{username}:#{password}@#{hostname}/#{database}"
+
+    source = insert(:source, user: insert(:user))
+    source_backend = insert(:source_backend, type: :postgres, config: %{url: url}, source: source)
+    repository_module = Repo.new_repository_for_source_backend(source_backend)
+    pipeline_name = Backends.via_source_backend(source_backend, Pipeline)
+    memory_buffer_pid = start_supervised!(MemoryBuffer)
```

Comment:
on a side note, we actually need to remove the MemoryBuffer to insert directly into broadway.